packages/babel-plugin-proposal-optional-chaining/test/fixtures/general/super-method-call/input.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
```

Comment:
Could you also add a `loose` test?  We can reuse this same `input.js` file, just copy it to a new folder and also copy https://github.com/babel/babel/blob/76cb2010e350a1ef69966d92c0d706e6115efca3/packages/babel-plugin-proposal-optional-chaining/test/fixtures/general/function-call-loose/options.json into the same fodler.

Reply:
Sure, added this.